packages/release/Cargo.lock

```diff
@@ -0,0 +1,591 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+version = 3
```

Comment:
This file should be removed - it gets created if you accidentally run `cargo make` in a package directory.